vanhouse/src/components/PostDetail.js

```diff
@@ -29,62 +28,6 @@ export default function PostDetail() {
     mapboxApiAccessToken: mapToken,
   });
 
-  // Comment function
-  const commentRef = useRef();
-  const [comments, setComments] = useState([
-    {
-      id: 0,
-      username: "Anon0",
-      comment:
-        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book",
-    },
```

Comment:
Did you pull the latest changes from project_3? These code are already in `project_3`, but you don't have them in your branch.

Reply:
@tyxxxx Here. You might need to merge or rebase your branch with your local `project_3`. Pull the newest changes to you local `project_3` before doing that.